src/system/process.py

```diff
@@ -66,9 +66,9 @@ def terminate(self) -> int:
                 try:
                     for item in proc.open_files():
```

Comment:
I believe this happened because we are checking all the process that is running on the host. The host has a number of processes running that may or may not be related to testing. Can you narrow down what process and open files we are checking?

Reply:
I think that's the properties of `psutil` to iterate all the running process though some of them are not related to our testing at all. I don't think there is a way to narrow down to specific process across platforms. Also some of the exception may be caused by race condition and we think we could pass on them since they are already outdated.

Reply:
You can get the process id from line 48. See if the process is subprocess of it? Rather than iterating over all the existing processes?

Reply:
@gaiksaya Basically in windows, according to my research, the `python.exe` was the process that was holding the file from removing which is not the `OpenSearch/Dashboards` process so that process id is not the one we want to check.

Reply:
The pid thing needs another research and bug fix as it was initially only designed for tar, not any other dist that can be handled by external means.

As of now the pid is not accurate unless is tar.